package.json

```diff
@@ -63,7 +63,7 @@
     "vnu-jar": "^21.10.12"
   },
   "dependencies": {
-    "@18f/identity-address-search": "^3.2.0",
+    "@18f/identity-address-search": "^3.4.0",
     "@18f/identity-build-sass": "^3.0.0",
```

Comment:
@gina-yamada I actually don't think I need to. The package is published with the correct version it's just that the tag was wrong. I was able to update the tag now things should be good to go 👍 

<img width="770" alt="image" src="https://github.com/user-attachments/assets/625bde08-84f5-4ce0-835e-e60de611665d">

Reply:
Awesome! I only saw the beta version in there when I commented. It would be great if you could share/add to the npm doc a link or the resource you used to update the tag. I will resolve this conversation now.